nixos/modules/services/misc/ethminer.nix

```diff
@@ -71,7 +71,7 @@ in
 
       maxPower = mkOption {
         type = types.int;
-        default = 115;
+        default = 113;
```

Comment:
Testing artifact?

Reply:
Anything over ~113 watts of power consumption is wasted (at least on the most popular 1070 nvidia cuda-based gpu cards).  By setting this default, we are potentially saving lots of wasted power usage. (each card unnecessarily consumes ~150watts of power if this is not set).   this setting only applies if the user selects cuda as the toolkit.

Reply:
I'm not sure how the change from 115 -> 113 can matter this much but it probably won't hurt

Reply:
yes, it would only matter if lots of people started mining with this package/module. it may just be me at the moment :)